docs_versioned_docs/version-ros2humble/ros/networking/ros2-networking.mdx

```diff
@@ -0,0 +1,242 @@
+there is no need to change from the default operation. Simple discovery generally allows for any ROS 2 and Fast DDS
+enabled device running the default settings to connect to the system with no additional configuration.
+
+:::tip
+
+If you want to run multiple independent simple discovery systems on the same network, you can segregate them by
+using different ROS Domain IDs.
+
+:::
+
+You may want or need to switch to discovery server for any of the following reasons:
+- If your network does not support multicasting (often true on school or corporate networks)
+- If you have a large system (several robots or an excess number of nodes/topics)
+- If you want to control which devices have access to which information
+- If you have a weak network (for example in cases of long range wireless connection)
+
+:::note
+
+Certain networks (such as school or corporate networks) may have additional restrictions that prevent even
+ROS 2 with discovery server from operating. In these situations consider meeting with the IT group or the network
+administrator to determine if ROS 2 is allowed to run on that network or if you can run the system on a separate
+dedicated network.
+
+:::
+
+## Discovery Server Configurations
+
+There are multiple different ways to configure discovery server depending on which ROS 2 devices need to connect.
+This section will go over some basic configurations and is generalized for use as an introduction. It does not
+describe all possible configurations. The `ROS_DISCOVERY_SERVER` environment variable, shown in the various diagrams,
+indicates which discovery servers a given robot is connected to. Additionally, communication will be referred to
+in terms of seeing ROS 2 topics although this is representative of all nodes, topics, action servers, etc. 
+
+:::note
+
+For all configurations, it is very important for all discovery servers in the system to be included in the `servers`
+list in the `robot.yaml` config. This is because each discovery server must have a unique ID number which
+is used both to launch the server and to connect to the server. This ID is automatically assigned by the Clearpath
+packages.
+
+:::
+
+### 1. Fully Connected {#fully-connected}
+
+This configuration is the simplest to set up using the `robot.yaml` config. It allows all robots to operate
+independently but also allows any ROS 2 device in the system to see all of the topics from all of the
+ROS 2 devices. This is done by having a local discovery server on each robot and pointing each ROS 2 device
+in the system at all of the discovery servers in the system.
+
+<center>
+  <figure>
+    <img
+      src={require("./img/ros2-fastdds-fully-connected-diagram.png").default}
+      width="500"
+    />
+  </figure>
+</center>
+
+This configuration is achieved by listing all of the computers' hostnames in the
+[`servers` section](../config/yaml/system.mdx#servers) on all of the robots. Including the remote PC allows
```

Comment:
Remove PC reference. "Computer" is preferred.
Refer to this comment for the background reasoning: 
https://github.com/clearpathrobotics/cpr-documentation/pull/81#discussion_r1116124370